src/store/index.js

```diff
@@ -13,6 +13,7 @@ export default new Vuex.Store({
     wmsAdministrativeBoundariesLayer: null,
     selectedLayer: null,
     selectedFeature: null,
+    currentUser: null,
```

Comment:
I could also create a separate store for this, and maybe avoid it being lost on refresh?

Reply:
Yeah, having a separate store for the user/authentication would be nice. Also if you have time left store the data in localstorage so data isn't being lost on refresh 👍

Reply:
I have separated the store, but for now won't use localstorage.